blog-service/2023-06-28-apps.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Druva Cyber Resilience (Apps)
```

Comment:
As this is a source, should it be collection? It looks like previous source release notes used this category https://help.sumologic.com/release-notes-service/2023/04/19/collection/

```suggestion
title: Druva Cyber Resilience (Collection)
```

Reply:
Yes, it should be collection. I just followed Apps for all the C2C release notes I created. I will update all the C2C release  note. Thanks Kim.